be/src/storage/lake/tablet_manager.cpp

```diff
@@ -2,58 +2,134 @@
+#include "util/uid_util.h"
 
 namespace starrocks::lake {
 
 TabletManager::TabletManager(GroupAssigner* group_assigner, int64_t cache_capacity)
         : _group_assigner(group_assigner), _metacache(new_lru_cache(cache_capacity)) {}
 
+// path $bucket:/$ServiceID/$GroupID/tbl_${TabletID}_${version}
+std::string TabletManager::tablet_meta_path(const std::string& group, int64_t tablet_id, int64_t verson) {
+    return fmt::format("{}/tbl_{:016X}_{:016X}", group, tablet_id, verson);
+}
+
+// trnslog path rule $Bucket:/$ServiceID/$GroupID/txn_${TabletID}_${TxnID}
+std::string TabletManager::txn_log_path(const std::string& group, int64_t tablet_id, int64_t txn_id) {
+    return fmt::format("{}/txn_{:016X}_{:016X}", group, tablet_id, txn_id);
+}
+
 Status TabletManager::create_tablet(const TCreateTabletReq& req) {
-    (void)req;
-    return Status::NotSupported("TabletManager::create_tablet");
+    // generate metapb
+    TabletMetadataPB tablet_metadata_pb;
+    tablet_metadata_pb.set_id(req.tablet_id);
+    tablet_metadata_pb.set_version(1);
+
+    // rowset metadata with version 1
+    auto rowset_metadata_pb = tablet_metadata_pb.add_rowsets();
+    rowset_metadata_pb->set_id(1);
+    rowset_metadata_pb->set_overlapped(false);
+    rowset_metadata_pb->set_data_size(0);
+    rowset_metadata_pb->set_num_rows(0);
+    tablet_metadata_pb.set_next_rowset_id(2);
+
+    // get shard group
+    ASSIGN_OR_RETURN(auto group_path, get_staros_shard_path(req.tablet_id));
+
+    // write tablet meta
+    auto meta_path = tablet_meta_path(group_path, tablet_metadata_pb.id(), tablet_metadata_pb.version());
+    ASSIGN_OR_RETURN(auto fs, FileSystem::CreateSharedFromString(meta_path));
+    ASSIGN_OR_RETURN(auto wf, fs->new_writable_file(meta_path));
+    RETURN_IF_ERROR(wf->append(tablet_metadata_pb.SerializeAsString()));
+    RETURN_IF_ERROR(wf->close());
+    return Status::OK();
 }
 
 StatusOr<Tablet> TabletManager::get_tablet(int64_t tablet_id) {
-    (void)tablet_id;
-    return Status::NotSupported("TabletManager::get_tablet");
+    ASSIGN_OR_RETURN(auto group_path, get_staros_shard_path(tablet_id));
+    return Tablet(this, group_path, tablet_id);
 }
 
 Status TabletManager::drop_tablet(int64_t tablet_id) {
-    (void)tablet_id;
-    return Status::NotSupported("TabletManager::drop_tablet");
+    vector<std::string> objects;
+    // get shard group
+    ASSIGN_OR_RETURN(auto group_path, get_staros_shard_path(tablet_id));
+    ASSIGN_OR_RETURN(auto fs, FileSystem::CreateSharedFromString(group_path));
+
+    auto scan_cb = [&objects, &tablet_id](std::string_view name) {
+        auto tablet_meta_prefix = fmt::format("tbl_{:016X}_", tablet_id);
+        auto txnlog_prefix = fmt::format("txn_{:016X}_", tablet_id);
+        if (!HasPrefixString(name, tablet_meta_prefix) && !HasPrefixString(name, txnlog_prefix)) {
+            // skip object for other tablet
+            return true;
+        }
+        objects.push_back(std::move(std::string(name)));
```

Comment:
```suggestion
        objects.emplace_back(name);
```

Reply:
`std::string` can be initialized with `std::string_view`, so you can construct a new `std::string` through `std::string_view` and append it to the end of `objects` via `emplace_back`.